### DISCUSSION TITLE
how to use the economy system?

### DISCUSSION THREAD

**Discussion by Jaz0314** [Tetlys/HeartsAndMindsPlus#11] [Q&A] [answered]
How does the economy system work? My group plays a lot of hearts and mind and when i saw your version i jumped on it. I'm trying to give something new since i still haven't figured out how to script new side missions for them.

**Comment by Tetlys** [comment]
Feel free to jump in discord and we can give you a run down :) My documentation rn isn't the best and my typing this whole thing out is gunna drive me crazy :'D https://discord.gg/7TXaQa7sEv

**Reply by Jaz0314** [reply]
Will do! I’m very interested in helping develop and test this. You’d probably be able to help learn some arms scripting as well.

**Reply by Tetlys** [reply]
For sure, always looking for help. Just hmu in discord :)

**Comment by Tetlys** [comment, marked as answer]
Basically:

* You must have the hearts and minds compat mod loaded

Kills, Objectives and side missions all produce $

When logged into admin, give build perms, save, relog (for others they just go to lobby and back)

Open build menu, can build whatever you want (the new build has preview buttons etc)

Salvage stations can be used to salvage vehicles from the enemy faction, the default price is set based on the zeus cost value (to save on config time)

All based on a combination of base hearts and minds, The ace fortify system, and my own code/ui to hook them together :) 

Can build within 100m of any FOB or HQ
